tools/flash_shelly.sh

```diff
@@ -330,7 +356,13 @@ function device_scan {
     probe_info $device $action $dry_run $mode
   else
     echo -e '\033[1mScanning for Shelly devices...\033[0m'
-    for device in $(timeout 2 dns-sd -B _http . | awk '/shelly/ {print $7}'); do
+    if [[ $arch == "Darwin" ]]; then
+      device_list=$(timeout 2 dns-sd -B _http . | awk '/shelly/ {print $7}' 2>/dev/null)
+    else
+      device_list=$(timeout 2 avahi-browse -pd local _http._tcp 2>/dev/null)
```

Comment:
avahi-browse -p -d local -t _http._tcp

so you don't need to use timeout

Reply:
ok

Reply:
https://github.com/mongoose-os-apps/shelly-homekit/pull/118/commits/800a6dd292b9146a53aecfffbaadec488c7cb641